stage1/init/kvm/hypervisor/hypervisor.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hypervisor
+
+// KvmHypervisor structure describes KVM hypervisor binary and its parameters
+type KvmHypervisor struct {
+	Bin          string
+	KernelParams []string
+}
+
+// SetParams sets common parameters passed to KVM hypervisors and debug parameters
+func (hv *KvmHypervisor) SetParams(isDebug bool) {
```

Comment:
@lucab Purpose of SetParams is to set only common parameters that we need to pass to the kernel every time we're configuring hypervisor (+debug). The rest of runtime configuration is passed through StartCmd which was built generic enough to support two hypervisors we have tested.